manifests/init.pp

```diff
@@ -146,6 +132,27 @@
     value    => $error_log,
   }
 
+  php::config { 'php-memory-limit-server':
+    file  => '/etc/php5/apache2/php.ini',
```

Comment:
I think we should make a new settings file for this in the location specified by `"${php::params::config_root_ini}"` as the code you're removing does because this path will not exist on CentOS.